charts/minecraft-bedrock/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: minecraft-bedrock
-version: 2.4.2
+version: 2.6.0
```

Comment:
There was already a 2.6.0, so it needs to be bumped a little bit more. 

(Sorry, I really don't like how the chart version management works out especially for concurrent changes)

Reply:
Bumped to 2.7.0 in https://github.com/itzg/minecraft-server-charts/pull/201/commits/735d257d64d053593b2d51ef9b1385d45b7b6075 but it didn't seem to let it progress so fell back to 2.6.0. I believe you have access to this PR if you can do the needfull?

Reply:
I'm confused, you then shifted it back down to 2.6.0 in  
https://github.com/itzg/minecraft-server-charts/commit/38c4c767198e02159e7ed23dce24b2bb36f85335

That commit you mentioned was fine -- yes, GitHub is needing me to manually approve the PR test runs.

Reply:
Just wasn't reflected as ok in UI (no change), maybe cache or something, not sure. What would you like for next action?

Reply:
Please re-commit the change to 2.7.0 and I will manually approve the PR build to test it.

Reply:
Bumped to 2.7.0.

Reply:
Complains it wants a bump.

Reply:
Yep needs another bump. There's been concurrent activity on the charts. Usually that's extremely rare. 

Is already 2.7.0 as of https://github.com/itzg/minecraft-server-charts/commit/ef6b865f3b6737d02be825ac1cb043d1b938d821

Reply:
Its now 2.8.0. Would be better to not do version changes by PR.

Reply:
> Would be better to not do version changes by PR.

I 1000% agree. I inherited this repo and just keep the lights on, but would be greatful if someone could solve the version mechanism.